### PR TITLE
Flux migration troubleshooting and IOx naming cleanup

### PR DESCRIPTION
**Pull request by sanderson**
Closes influxdata/DAR#387

- Adds an obscure error and resolution to Flux migration script doc 
- Some IOx naming cleanup (still more to do)
- Renamed some IOx-named files

---

- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/cloud-serverless/guides/migrate-data/migrate-1x-to-serverless.md [hunk 1]
```diff
@@ -10,7 +10,8 @@ menu:
     parent: Migrate data
 weight: 103
 aliases:
-  - /influxdb/cloud-serverless/write-data/migrate-data/migrate-1x-to-iox
+  - /influxdb/cloud-serverless/write-data/migrate-data/migrate-1x-to-iox/
+  - /influxdb/cloud-serverless/guides/migrate-data/migrate-1x-to-iox/
```
Comment (jstirnaman) [inline content/influxdb/cloud-serverless/guides/migrate-data/migrate-1x-to-serverless.md:14]: I was just making this same edit.

File: content/influxdb/v2/write-data/migrate-data/migrate-cloud-to-oss.md [hunk 2]
```diff
@@ -373,3 +382,15 @@ task will fail.
 - Update the `migration.batchInterval` setting in your migration task to use
   a smaller interval. Each batch will then query less data and take less time
   to return results.
+
+### Batch size is too large
+
+If your batch size is too large, the task will return an error similar to:
```
Comment (jstirnaman) [inline content/influxdb/v2/write-data/migrate-data/migrate-cloud-to-oss.md:388]:
```suggestion
If your batch size is too large, the task returns an error similar to the following:
```

File: content/influxdb/v2/write-data/migrate-data/migrate-cloud-to-oss.md [hunk 3]
```diff
@@ -373,3 +382,15 @@ task will fail.
...
+
+```
+internal error: error calling function "metadata" @97:1-97:11: error calling function "findRecord" @67:32-67:69: wrong number of fields
+```
+
+**Possible solutions**:
+- Update the `migration.batchInterval` setting in your migration task to use
+  a smaller interval. Each batch will then query less data.
```
Comment (jstirnaman) [inline content/influxdb/v2/write-data/migrate-data/migrate-cloud-to-oss.md:396]:
```suggestion
- Update the `migration.batchInterval` setting in your migration task to use
  a smaller interval and retrieve less data per batch.
```

**Review by jstirnaman — APPROVED** [review]
👍 A couple of suggestions.

File: content/influxdb/cloud-serverless/guides/migrate-data/migrate-tsm-to-serverless.md [hunk 4]
```diff
@@ -397,3 +405,15 @@ task will fail.
...
+
+If your batch size is too large, the task will return an error similar to:
+
+```
+internal error: error calling function "metadata" @97:1-97:11: error calling function "findRecord" @67:32-67:69: wrong number of fields
+```
+
+**Possible solutions**:
+- Update the `migration.batchInterval` setting in your migration task to use
+  a smaller interval. Each batch will then query less data.
```
Comment (sanderson) [inline content/influxdb/cloud-serverless/guides/migrate-data/migrate-tsm-to-serverless.md:419]:
~~~suggestion
If your batch size is too large, the task returns an error similar to the following:

```
internal error: error calling function "metadata" @97:1-97:11: error calling function "findRecord" @67:32-67:69: wrong number of fields
```

**Possible solutions**:
- Update the `migration.batchInterval` setting in your migration task to use
  a smaller interval and retrieve less data per batch.
~~~

File: content/influxdb/cloud/write-data/migrate-data/migrate-cloud-to-cloud.md [hunk 5]
```diff
@@ -385,17 +392,30 @@ task will fail.
...
+
+If your batch size is too large, the task will return an error similar to:
+
+```
+internal error: error calling function "metadata" @97:1-97:11: error calling function "findRecord" @67:32-67:69: wrong number of fields
+```
+
+**Possible solutions**:
+- Update the `migration.batchInterval` setting in your migration task to use
+  a smaller interval. Each batch will then query less data.
```
Comment (sanderson) [inline content/influxdb/cloud/write-data/migrate-data/migrate-cloud-to-cloud.md:405]:
~~~suggestion
If your batch size is too large, the task returns an error similar to the following:

```
internal error: error calling function "metadata" @97:1-97:11: error calling function "findRecord" @67:32-67:69: wrong number of fields
```

**Possible solutions**:
- Update the `migration.batchInterval` setting in your migration task to use
  a smaller interval and retrieve less data per batch.
~~~

**Pull request merged**